src/models.mysql/init-models.js

```diff
@@ -0,0 +1,26 @@
+var DataTypes = require("sequelize").DataTypes;
+var _Event = require("./Event");
+var _Participant = require("./Participant");
+var _User = require("./User");
+
+function initModels(sequelize) {
+  var Event = _Event(sequelize, DataTypes);
+  var Participant = _Participant(sequelize, DataTypes);
+  var User = _User(sequelize, DataTypes);
+
+  Event.belongsTo(User, { foreignKey: "creator_id"});
+  User.hasMany(Event, { foreignKey: "creator_id"});
+  Participant.belongsTo(User, { foreignKey: "user_id"});
+  User.hasMany(Participant, { foreignKey: "user_id"});
+  Participant.belongsTo(Event, { foreignKey: "event_id"});
+  Event.hasMany(Participant, { foreignKey: "event_id"});
+
```

Comment:
there is such method as `belongsToMany` which might be better here in some cases.